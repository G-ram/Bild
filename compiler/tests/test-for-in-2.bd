a = {"a" : 0,"b" : 3,"c" : 5, "d" : 7};
for(b in a){
  print(b + " : " + a[b]);
}
