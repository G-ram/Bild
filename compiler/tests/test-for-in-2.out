a : 0
b : 3
c : 5
d : 7
